torchmultimodal/transforms/flava_transform.py

```diff
@@ -200,9 +204,30 @@ def __call__(self, img):
 
 
 class MaskedImageModelingTransform:
+    """Masked image modeling transform
+    Randomly masks patches in an image based on scheme from Beit
+    https://arxiv.org/pdf/2106.08254.pdf
+
+    Args:
+        is_train (bool): whether transform is applied during training or not
```

Comment:
For arguments with default values, it's typical to add a statement at the end of each docstring. For example:

```
is_train (bool): whether transform is applied during training or not. Defaults to ``True``.
```